bindings/kafka/metadata.yaml

```diff
@@ -0,0 +1,186 @@
+# yaml-language-server: $schema=../../component-metadata-schema.json
+schemaVersion: v1
+type: bindings
+name: kafka
+version: v1
+status: stable
+title: "Apache Kafka"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-bindings/kafka/
+binding:
+  output: true
+  input: true
+  operations:
+    - name: create
+      description: "Publish a new message in the topic."
+authenticationProfiles:
+  - title: "OAuth Authentication"
+    description: |
+      Authenticate using OAuth.
+    metadata:
```

Comment:
This auth profile misses "authType" and must say in this profile it must be set to "oidc"